release/ml_user_tests/tune_rllib/app_config.yaml

```diff
@@ -0,0 +1,26 @@
+base_image: "anyscale/ray-ml:pinned-nightly-py37-gpu"
```

Comment:
so I just fixed a minor issue where the long_running_tests were using a really dated app_config to run rllib stacks.
should we not create another copy of app_config for this, and simply point to: ../../rllib_tests/app_config.yaml ?

Reply:
Good point, I’ll do that in a follow up PR. Just going to merge this for now to unblock @alanwguo